.github/workflows/phpunit-oci.yml

```diff
@@ -53,7 +56,7 @@ jobs:
               - 'composer.lock'
 
   phpunit-oci:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
This is not needed anymore and can be changed upstream (but let's wait until 24.04 is served, so we can see if that also works on latest)